src/Simulation/Simulators.Tests/TestProjects/UnitTests/UnitTests.csproj

```diff
@@ -15,10 +15,10 @@
     <ProjectReference Include="..\..\..\Simulators\Microsoft.Quantum.Simulators.csproj" />
     <ProjectReference Include="..\Library1\Library1.csproj" />
     <ProjectReference Include="..\Library2\Library2.csproj" />
+    <ProjectReference Include="..\..\..\..\Xunit\Microsoft.Quantum.Xunit.csproj" />
```

Comment:
Actually I believe we don't need that reference - if you could remove it.

Reply:
Sounds good, will do!